Add interfaces/frame-timing.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/frame-timing.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389

stephenmcgruer commented:
Decision task failed, cycling the PR to see if it works next time. There are no tests in https://wpt.fyi/results/?diff&filter=ADC&run_id=416700007&run_id=403440006, so this probably needs a test added.

stephenmcgruer commented:
Started implementing tests, and discovered that nobody seems to implement this (according to https://developer.mozilla.org/en-US/docs/Web/API/PerformanceFrameTiming). Given that, do we really want this imported @foolip ?

stephenmcgruer commented:
Ping @igrigorik to check - do you know the implementation status of `PerformanceFrameTiming` - is any browser currently working on implementing it?

autofoolip committed:
Add interfaces/frame-timing.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/frame-timing.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389
